pages/index.js

```diff
@@ -1,40 +1,77 @@
-import Heading from "@/Header";
 import InterviewCard from "@/components/InterviewCard";
 import { interviews } from "@/public/interviews";
 import SearchInput from "@/components/Search/SearchInput";
 import SearchResults from "@/components/Search/SearchResults";
 import { useState } from "react";
 import Footer from "@/components/Footer";
+import styled from "styled-components";
 
+const INTERVIEWS_PER_PAGE = 2;
```

Comment:
Nice 👍 Very good practice.

Reply:
Thank you, but it was Pete's suggestion, not taking credit :)

Reply:
Good job listening to Pete, then! 😂